### DISCUSSION TITLE
Explanation for parent_id Field in Version 2

### DISCUSSION THREAD

**Discussion by IHEEB22** [jaredhendrickson13/pfsense-api#459] [Q&A]
In some endpoints of the Version 2 , a new field, parent_id, has been introduced. However, there seems to be a lack of clear documentation explaining its purpose and usage. This ambiguity can lead to confusion for developers trying to integrate with the API. It's essential to provide a concise explanation of the parent_id field to enhance usability and developer experience.

**Comment by jaredhendrickson13** [comment]
Sorry for not seeing this one earlier. In case anyone else comes across this, there is a section outlining the `parent_id` field in the docs site that should help: https://pfrest.org/WORKING_WITH_OBJECT_IDS/#object-parent_id-field

The `parent_id` is also documented is under the `schema` of applicable endpoints in the Swagger doc.
